packages/web-components/fast-components-msft/src/tree-item/tree-item.styles.ts

```diff
@@ -0,0 +1,235 @@
+import { css } from "@microsoft/fast-element";
+import {
+    disabledCursor,
+    display,
+    focusVisible,
+    forcedColorsStylesheetBehavior,
+} from "@microsoft/fast-foundation";
+import { SystemColors } from "@microsoft/fast-web-utilities";
+import {
+    accentForegroundRestBehavior,
+    heightNumber,
+    neutralFillStealthActiveBehavior,
+    neutralFillStealthHoverBehavior,
+    neutralFillStealthRestBehavior,
+    neutralFillStealthSelectedBehavior,
+    neutralFocusBehavior,
+    neutralFocusInnerAccentBehavior,
+    neutralForegroundActiveBehavior,
+    neutralForegroundRestBehavior,
+} from "../styles/index";
+
+export const TreeItemStyles = css`
+    ${display("block")} :host {
+        contain: content;
+        position: relative;
+        outline: none;
+        color: var(--neutral-foreground-rest);
```

Comment:
each behavior now allows access to the CSS custom property variable from `Behavior.var, which you can interpolate into the stylesheet.

Reply:
done